source/standards/dns-hosting.html.md.erb

```diff
@@ -1,33 +1,58 @@
 ---
 title: How to manage DNS records for your service
-last_reviewed_on: 2024-02-21
+last_reviewed_on: 2025-02-24
 review_in: 12 months
 ---
 
 # <%= current_page.data.title %>
 
 To ensure users can connect to your service, you'll need accurate Domain Name System (DNS) records and a DNS provider to supply your DNS nameservers.
 
-You can find information on choosing where to host your DNS and how to request nameserver delegation at GDS in the [Service Manual][].
+For services [hosted on AWS](./hosting.html) you should use [Amazon Route 53][] as a cloud DNS web service.
 
-As part of your DNS strategy and planning you need to ensure:
+> [!INFO]
+> In the past some services have dual-hosted using [Google Cloud DNS][] for high nameserver availability.
+> However, this adds complexity and risk so you should not do this without a careful analysis of how it could affect overall reliability.
 
-* your DNS record configuration is reproducible (for example, the DNS records are version controlled or autogenerated)
-* you consider DNS availability as part of overall service availability.
+You can read more about [service domains and DNS][], including how to apply for a domain name for citizen-facing services, in the Service Manual.
 
-For example, `publishing.service.gov.uk` is published to both Amazon Route 53 and Google Cloud DNS to remove vendor as a single point of failure.
-This means that even if AWS has a total outage, users can access parts of GOV.UK by using Google Cloud DNS to resolve addresses, and viewing content that is available in the CDN cache.
+[Service Manual]: https://www.gov.uk/service-manual/technology/get-a-domain-name#choose-where-youll-host-your-dns
+[Amazon Route 53]: https://aws.amazon.com/route53/
+[Google Cloud DNS]: https://cloud.google.com/dns/
+[service domains and DNS]: https://www.gov.uk/service-manual/technology/get-a-domain-name
 
-Note that if your service availability relies on a single vendor, there is less benefit to deploying DNS to multiple vendors.
+## Production and non-production domain names
 
-When you implement your DNS strategy you should consider using:
+Your non-production domains should be subdomains of your production domain.
 
-* [Amazon Route 53][] as a cloud DNS web service
-* [Google Cloud DNS][] for high nameserver availability
+For example, if you service can be found at `myservice.service.gov.uk`, then you might also have Route 53 records for your other environments, such as:
 
-You can read more about [service domains and DNS][] in the Service Manual.
+* `staging.myservice.service.gov.uk`
+* `dev.myservice.service.gov.uk`
```

Comment:
```suggestion
* `staging.myservice.service.gov.uk`
* `dev.myservice.service.gov.uk`

> [!TIP]
> In this example it's important to avoid setting cookies for the entire domain `myservice.service.gov.uk`.
> Always follow the
> [OWASP guidance on cookie domains](https://cheatsheetseries.owasp.org/cheatsheets/Session_Management_Cheat_Sheet.html#domain-and-path-attributes)
> such as to avoid using the `Domain` cookie attribute.
> You can also consider using a separate domain for your production service, such as `www.myservice.service.gov.uk`.```